pkg/actors/placement/placement.go

```diff
@@ -128,6 +130,7 @@ func NewActorPlacement(opts internal.ActorsProviderOptions) internal.PlacementSe
 		closeCh:           make(chan struct{}),
 		resiliency:        opts.Resiliency,
 		virtualNodesCache: hashing.NewVirtualNodesCache(),
+		namespace:         security.CurrentNamespace(),
```

Comment:
Please pass as option as this function uses env var which can be awkward for testing.